core/sail/federation/src/main/java/org/eclipse/rdf4j/sail/federation/optimizers/RepositoryBloomFilter.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Minor thing, but you're using the wrong copyright header. For new files, Aduna should not be mentioned. See the copyright header template in eclipse-settings/codetemplates.xml .